ui/src/mixins/virtual-scroll.js

```diff
@@ -434,7 +434,7 @@ export default {
             h('tr', [
               h('td', {
                 style: { [paddingSize]: `${this.virtualScrollPaddingBefore}px` },
-                attrs: { colspan: '100%' }
+                attrs: { colspan: this.columnCount || '100%' }
```

Comment:
change this to `attrs: { colspan: this.columnCount === void 0 ? '100%' : this.columnCount }`